src/utils.cc

```diff
@@ -436,28 +436,18 @@ void free_memory(nagios_macros* mac) {
   // Free memory allocated to downtimes.
   downtimes::downtime_manager::instance().clear_scheduled_downtimes();
 
-  // Free memory for the high priority event list.
-  for (timed_event_list::iterator it(timed_event::event_list_high.begin()),
-       end(timed_event::event_list_high.end());
-       it != end;) {
```

Comment:
I think this is leftover from another commit

Reply:
This just because the branch was build on top of the one with segfault already merged :smile:

Reply:
The rebase solved this